cli/tests/unit/fetch_test.ts

```diff
@@ -634,8 +615,8 @@ function bufferServer(addr: string): Buffer {
     await Promise.all([p1, p2]);
     conn.close();
     listener.close();
+    return buf;
   });
-  return buf;
```

Comment:
I'm a little confused... is there a reason why this wasn't done before and the code was this way?

I think previously the `conn.write` could complete before the `buf.readFrom` causing the tests to read from the buffer and it being empty.

Reply:
I believe it was just a mistake in code and your version if the proper one